vm-setup/roles/common/defaults/main.yml

```diff
@@ -14,6 +14,7 @@ default_vcpu: 4
 num_nodes: 2
 extradisks: false
 virtualbmc_base_port: 6230
+sushy_port: 8000
```

Comment:
this should be defined in vm-setup/roles/firewall/defaults/main.yaml , if in common, ansible does not find it :
`The task includes an option with an undefined variable. The error was: 'sushy_port' is undefined\n`

Reply:
> this should be defined in vm-setup/roles/firewall/defaults/main.yaml , if in common, ansible does not find it :
> `The task includes an option with an undefined variable. The error was: 'sushy_port' is undefined\n`

thanks, I was hoping I'd be able to use it from ironic_nodes.json.j2 by putting in common/defaults/main.yml but will put it back into the firewall role